src/pages/index.js

```diff
@@ -1,325 +1,253 @@
+import Link from '@docusaurus/Link'
 
-class HomeSplash extends React.Component {
-  render() {
-    const { siteConfig } = this.props
-    const { baseUrl } = siteConfig
+export default function Index(props) {
+  const { config: siteConfig } = props
 
-    const SplashContainer = (props) => (
-      <div className='homeContainer'>
-        <div className='homeSplashFade'>
-          <div className='wrapper homeWrapper'>{props.children}</div>
-        </div>
-      </div>
-    )
-
-    const Logo = (props) => (
-      <img
-        src={props.img_src}
-        alt='Web Monetization Icon'
-        style={{ width: '120px' }}
-      />
-    )
-
-    const ProjectTitle = () => (
-      <div className='projectTitle'>
-        <h2>{siteConfig.title}</h2>
-        <small>{siteConfig.tagline}</small>
-      </div>
-    )
-
-    return (
-      <SplashContainer>
-        <Logo img_src={`${baseUrl}img/wm-icon-animated.svg`} />
-        <img src={`${baseUrl}img/grey_wm_logo.svg`} className='bgLogo' alt='' />
-        <div className='inner'>
-          <ProjectTitle siteConfig={siteConfig} />
-        </div>
-      </SplashContainer>
-    )
-  }
-}
-
-export default class Index extends React.Component {
-  render() {
-    const { config: siteConfig, language = '' } = this.props
-    const { baseUrl, docsUrl } = siteConfig
-    const docsPart = `${docsUrl ? `${docsUrl}/` : ''}`
-    const langPart = `${language ? `${language}/` : ''}`
-    const docUrl = (doc) => `${baseUrl}${docsPart}${langPart}${doc}`
-
-    const SpashFooter = () => {
-      return (
-        <div className='splashFooter'>
-          <div>
-            Web Monetization is being proposed as a W3C standard at&nbsp;
-            <a
-              href='https://discourse.wicg.io/t/proposal-web-monetization-a-new-revenue-model-for-the-web/3785'
-              target='_blank'
-              rel='noreferrer noopener'
-            >
-              the Web Platform Incubator Community Group
-            </a>
-            .
-          </div>
-        </div>
-      )
-    }
-
-    const TextBlock = (props) => (
-      <div className='contentBlock'>
-        <div className='contentContainer'>{props.children}</div>
-      </div>
-    )
-
-    const Motivation = () => (
-      <TextBlock>
-        <h2>Motivation</h2>
-        <p>
-          The ability to transfer money has been a long-standing omission from
-          the web platform. As a result, the web suffers from a flood of
-          advertising and corrupt business models. Web Monetization provides an
-          open, native, efficient, and automatic way to compensate creators, pay
-          for API calls, and support crucial web infrastructure.
-        </p>
-      </TextBlock>
-    )
-
-    const WhyNow = () => (
-      <TextBlock>
-        <h2>Why Now?</h2>
-        <p>
-          Until recently, there hasn't been an open, neutral protocol for
-          transferring money.{' '}
+  const CardList = (props) => {
```

Comment:
Simplified the component structure to just having `CardList` to deal with the card data pulled from docusaurus.config.js and `PrCta` which is repeated throughout the sections asking folks to open a PR for proposed suggestions.